### PR TITLE
Add missing whitespaces

### PR DESCRIPTION
**Pull request by Razzeee**
Spotted some missing whitespaces.

Not sure how to guard this against them being removed by a whitespace trim IDE integration.

### CODE REVIEW

**Comment by aaronraimist** [comment]
Is there not enough room to move it to one line or reorganize it in a way where whitespace won't be trimmed?

**Comment by Razzeee** [comment]
Really depends on your linting options for js I guess, as I did this as a drive by, I'm not sure how the repo is configured :)

File: gatsby/src/pages/faq.js [hunk 1]
```diff
@@ -117,8 +117,8 @@ const Faq = ({data}) => {
 <div className="question">
 <h4 id="how-is-matrix.org-funded%3F"><a className="permalink" href="#how-is-matrix.org-funded%3F" aria-hidden="true">&#128279;</a> How is Matrix.org funded?</h4>
 <p>Matrix.org is currently funded by the community, through a
-combination of community support (via
-<a href="https://patreon.com/matrixdotorg">Patreon</a>,
+combination of community support (via 
+<a href="https://patreon.com/matrixdotorg">Patreon</a>, 
 <a href="https://liberapay.com/matrixdotorg">Liberapay</a>, Bitcoin and Ethereum),
```
Comment (aaronraimist) [inline gatsby/src/pages/faq.js:122]:
```suggestion
combination of community support (via
<a href="https://patreon.com/matrixdotorg"> Patreon</a>,
<a href="https://liberapay.com/matrixdotorg"> Liberapay</a>, Bitcoin and Ethereum),
```

**Comment by aaronraimist** [comment]
Well that isn't right but anyway you could reorganize it so whitespace stripping doesn't matter

File: gatsby/src/pages/faq.js (hunk 1 above)
Comment (SuperSandro2000) [reply gatsby/src/pages/faq.js:122]: this would make the space clickable which is not ideal.

**Comment by benparsons** [comment]
FYI this will be replaced with markdown when I have a chance, for now I'm commiting it per @Razzeee

**Pull request merged**